tutorial/source/RSA-hyperbole.ipynb

```diff
@@ -5,7 +5,6 @@
    "metadata": {},
    "source": [
     "# Understanding Hyperbole using RSA\n",
-    "(This tutorial written by Noah Goodman and Eli Bingham, following Kao, Wu, Bergen, and Goodman (2014).)\n",
```

Comment:
i still think we should attribute authors in contributed tutorials, but we can revive this discussion after 0.3